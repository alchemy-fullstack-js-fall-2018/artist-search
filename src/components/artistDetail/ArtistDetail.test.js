// import React from 'react';
// import ArtistDetail from './ArtistDetail';
// import { shallow } from 'enzyme';

// describe('artistDetail', () => {
//     let wrapper;

//     beforeEach(() => {
//         wrapper = shallow(<ArtistDetail />);
//     });

//     it('renders a artistDetail component', () => {
//         expect(wrapper).toMatchSnapshot();
//     });
// });
