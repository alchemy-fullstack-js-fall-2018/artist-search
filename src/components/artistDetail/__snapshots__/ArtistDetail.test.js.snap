// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`artistDetail renders a artistDetail component 1`] = `undefined`;
